src/frontend/src/instance/standalone.rs

```diff
@@ -107,7 +107,7 @@ impl Datanode for RegionInvoker {
     }
 }
 
-pub(crate) struct StandaloneTableMetadataCreator {
+pub struct StandaloneTableMetadataCreator {
```

Comment:
Nit: Adding some tests is better if you use this in the outside crate. The upstream changes might break the downstream code unintentionally. Or log it in the issues and leave it for future work.